c++/src/capnp/rpc.c++

```diff
@@ -3732,6 +3819,24 @@ public:
     KJ_UNREACHABLE;
   }
 
+  kj::Promise<void> sendRealtime(kj::Own<OutgoingRpcMessage> message) override {
+    KJ_SWITCH_ONEOF(state) {
+      KJ_CASE_ONEOF(ignored, Running) {
+          if (isReady()) {
+            message->sendRealtime();
+            return kj::READY_NOW;
+          } else if (!hasLoggedDroppedRealtimeMessage) {
+            KJ_DBG("Dropping realtime message(s) because of link congestion");
+            hasLoggedDroppedRealtimeMessage = true;
```

Comment:
@kentonv: I don't really like this, but I wanted to show a proposition for logging it. But I guess it would be better to just silently drop the message here, especially given that the returned promise will show that the message got dropped, so the app can log it if necessary.

What do you think?

Reply:
Nevermind, I'm pretty sure that logging here was a bad idea. I removed it.